### PR TITLE
Add Homebridge

### LINKED ISSUE #3464: Homebridge
**Name:** Homebridge
**Website:** https://homebridge.io
**Alexa rank:** 622,544
  

**Official resources for icon and color:**  https://github.com/homebridge/branding
  
  

Homebridge is a popular smart home platform that provides HomeKit support for non-HomeKit-native devices.

### PR DESCRIPTION
**Pull request by fbernhart**
homebridge

Closes #3464 
**Alexa rank:** only [673,374](https://www.alexa.com/siteinfo/homebridge.io), BUT [16.6k stars on GitHub](https://github.com/homebridge/homebridge)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Source: [This SVG file](https://github.com/homebridge/branding/blob/6ef3a1685e79f79a2ecdcc83824e53775ec0475d/logos/homebridge-silhouette-round-black.svg), from their branding repo.

Color: [The colorized SVG logo](https://github.com/homebridge/branding/blob/6ef3a1685e79f79a2ecdcc83824e53775ec0475d/logos/homebridge-color-round.svg) has a color gradient - from `#491F59` to `#57277C`. I've used the midpoint `#50236B`, calculated through: https://meyerweb.com/eric/tools/color-blend/#491F59:57277C:1:hex

### CODE REVIEW

**Comment by hjdhjd** [comment]
#491F59 is what we use mostly for Homebridge branding / badging. I would not use the midpoint...it’s not true to the overall Homebridge branding.

Hope you have better luck getting this approved than my submission. 😄 #3542 and #3616, which was @brandawg93 to take a stab at it after mine was rejected.

**Comment by fbernhart** [comment]
@hjdhjd Thanks for your feedback. As you're speaking of "we", I was wondering if you're a maintainer of https://github.com/homebridge/homebridge or something like this. If so, I'm happy to change it to `#491F59`.

Otherwise, could you at least give some examples where `#491F59` is used?

**Comment by hjdhjd** [comment]
[Homebridge Reddit](https://www.reddit.com/r/homebridge/) as an example.

I’d use the Homebridge branding site as your guide, and when in doubt, open an issue and ask for guidance rather than assume. I don’t maintain the official branding, but @brandawg93 and I both maintain several of the more popular plugins for Homebridge and are actively involved in the community.

Your midpoint looks plainly off (to my eye) of what the color scheme is for the Homebridge logo and branding - it should be toward the darker end of the range, not the lighter end. In fairness, you shouldn’t take my word for it, you should get the official guidance.

When I’d approached Homebridge branding for permission to create a simple-icon, I didn’t ask a specific question on color, but that’s because I thought the color was obviously closer to the dark end of the spectrum. :smile:

TL;DR: open an issue at https://github.com/homebridge/branding and get official response. Otherwise, we’re all just giving our opinions rather than trying to converge on a consistent, and authoritative, answer. :smile:

**Comment by fbernhart** [comment]
@Supereg and @oznu seem to be maintainers of homebrew. Can one of you help us out here and let us know the preferred color? 

Which one represents homebrew the best?  `#491F59`, `#57277C` or `#50236B` (midpoint)?

**Comment by oznu** [comment]
@fbernhart #491F59 looks good.

thank you!

**Comment by fbernhart** [comment]
Thanks for the feedback @oznu! 👍🏼

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @fbernhart 🏠🌉
